backend/app/user/controller.py

```diff
@@ -17,8 +18,15 @@
 async def get_user_route(user_id: ObjectID,
                          db: AsyncIOMotorClient = Depends(get_database)) -> UserInResponse:
     """Get user information for logged in user"""
-    # TODO fetch user functionality
-    return {"success": True}
+    if(ObjectId.is_valid(user_id) == False):
+        raise HTTPException(
+            status_code=400, detail='Not a valid UserID')
```

Comment:
Not required, since `user_id: ObjectID` already verifies valid ObjectIds

Reply:
Doing this causes Internal Server Error when I'm using an invalid id

Reply:
Let me check

Reply:
Something similar has been done in Pull #36

Reply:
@Pk13055 Fixes the issue. I'll update everything and push

Reply:
Perfect! Thanks for the quick response.